### PR TITLE
Fix some small typos 😉

### PR DESCRIPTION
**Pull request by kutyel**
Hi Dmitrii!

Thanks very much for this useful post, I learnt a lot from it!
There are a few grammar mistakes that I'd like to help you correct, feel free to discard this PR! 😉 

Keep up the good work! 💪

### CODE REVIEW

File: posts/2019-07-29-recordwildcards.md [hunk 1]
```diff
@@ -167,10 +166,10 @@ fields like in the code below:
 defaultUser :: User
 defaultUser =
     let userName = "Ivan"
-    in User{..}
+     in User{..}
```
Comment (chshersh) [inline posts/2019-07-29-recordwildcards.md]: I think this one is not a typo. There should be 4 spaces before each line if I counted properly :slightly_smiling_face: 
```suggestion
    in User{..}
```

**Review by chshersh — APPROVED** [review]
Hi @kutyel! Thanks a lot for your fixes! Great work, I really appreciate it :heart: 

I have only one minor comment. Otherwise, look fantastic :100: 

And thanks for your kind words about the post! I'm glad you found it interesting :blush:

File: posts/2019-07-29-recordwildcards.md (hunk 1 above)
Comment (kutyel) [reply posts/2019-07-29-recordwildcards.md]: Yes, sorry, this is my ormolu OCD acting on me 😂

**Comment by kutyel** [comment]
Maybe you can consider adding the `hacktoberfest-accepted` before merging? 🙏

**Comment by chshersh** [comment]
@kutyel Done! And thanks for your fixes again :blush:

**Pull request merged**